configs/vitest.config.e2e.ts

```diff
@@ -1,8 +1,11 @@
 import path from "node:path";
 import {defineProject} from "vitest/config";
 
-export const e2eProject = defineProject({
+export const e2eMinimalProject = defineProject({
```

Comment:
@nazarhussain what do you think about keeping it in variable name, it is anyways only used in one place and might help clarify when looking at the config